include/erlcloud_ddb2.hrl

```diff
@@ -16,16 +16,78 @@
 
 -type date_time() :: number().
 -type global_table_status() :: creating | active | deleting | updating.
+-type replica_status() :: creating | active | deleting | active.
```

Comment:
Should be the following. I will update this and the undynamizer function.

`-type replica_status() :: creating | creation_failed | updating | deleting | active.`